utils/cli-utils/Cargo.toml

```
@@ -0,0 +1,23 @@
+[package]
+name = "cli-utils"
+version = "2.0.0-dev"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0"
```

Utils are Apache-2.0 in the new scheme
```suggestion
license = "Apache-2.0"
```